### ISSUE TITLE
Theme sections not loading in WP 4.8.1

### ISSUE THREAD

**Issue by tomeli5n** [themejack/fiera#9]
Hi,

I tried  fiera-1.0.2 release in a fresh install of WP 4.8.1 under php 5.6.31

The theme loads but no section is shown under the customizer.

![image](https://user-images.githubusercontent.com/7417323/30462203-8d5c2902-999a-11e7-8c03-a28349bd73f0.png)

There in not error in the chrome console...
Should I add something more?

Thanks in advance! :)

**Comment by anteprimorac** [comment]
Hi @tomeli5n ,
did you set page for front page? Did you add any section into that page?

**Comment by tomeli5n** [comment]
Hi @anteprimorac 
Yes, I was able to solve the problem.
I had set the page, but I was expecting the sections appear in the customizer.

Now I learn that the sections are seted in the page editor.

Thanks! very nice theme

**Comment by anteprimorac** [comment]
Glad to hear you like it. I am closing this issue.
